### PR TITLE
Updated installation language files

### PR DESCRIPTION
**Pull request by Bakual**
Can be merged once checks passed.

### CODE REVIEW

File: installation/language/lv-LV/lv-LV.xml [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
-	<author>Latvian Translation Team</author>
+	<name>Latvian (Lv)</name>
+	<version>3.7.4</version>
+	<creationDate>2017. gada jūlijs</creationDate>
+	<author>Latvian Joomla translation team</author>
 	<copyright>Copyright (C) 2005 - 2017 Open Source Matters. All rights reserved.</copyright>
 	<license>GNU General Public License version 2 or later; see LICENSE.txt</license>
 	<files>
-		<filename>lv-LV.ini</filename>
+		<filename>lv-LV.lib_joomla.sys.ini</filename>
```
Comment (infograf768) [inline installation/language/lv-LV/lv-LV.xml]: wrong

File: installation/language/lv-LV/lv-LV.xml [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
-	version="3.6"
+	version="3.7"
 	client="installation">
-	<name>Latvian (LV)</name>
-	<version>3.4.4</version>
-	<creationDate>August 2015</creationDate>
-	<author>Latvian Translation Team</author>
+	<name>Latvian (Lv)</name>
+	<version>3.7.4</version>
+	<creationDate>2017. gada jūlijs</creationDate>
```
Comment (infograf768) [inline installation/language/lv-LV/lv-LV.xml]: should be english date

**Comment by Bakual** [comment]
Updated

**Comment by infograf768** [comment]
rtc. merging<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17143">issues.joomla.org/tracker/joomla-cms/17143</a>.</sub>

**Pull request merged**